ckan_cloud_operator/providers/ckan/instance/manager.py

```diff
@@ -39,14 +41,23 @@ def create(instance_type, instance_id=None, instance_name=None, values=None, val
     if values_id and values_id != instance_id:
         logs.warning(f'changing instance id in spec from {values_id} to the instance id {instance_id}')
     values['id'] = instance_id
+
+    use_cloud_storage = config_manager.get('use-cloud-native-storage', secret_name=CONFIG_NAME)
+    values['useCloudStorage'] = use_cloud_storage
```

Comment:
why override this value and not use the value in the spec?

Reply:
If this value is true, we can verify`use-cloud-native-storage` is indeed true.

Reply:
done